e2e-tests/cypress/integration/common/cookies.js

```diff
@@ -57,13 +56,11 @@ When('the user visits the service', () => {
 });
 
 Then('not necessary cookies are disabled', () => {
-  const expectedCookiePolicy = JSON.stringify(DEFAULT_COOKIE_POLICY);
-
   cy.getCookies()
-    .should('have.length', 2)
+    .should('have.length', 1)
     .then((cookies) => {
       expectExpressSessionCookieIsDefined(cookies);
-      expectCookiePolicy(cookies, expectedCookiePolicy);
+      expectCookieIsNotDefined(cookies, 'cookie_policy');
```

Comment:
feels weird to remove this but it's in line with the AC. This is going right back in with the JS banner variant of this feature.